Sources/Paywall/Cache/Cache.swift

```diff
@@ -15,9 +15,16 @@ final class Cache {
   static let cacheDirectoryPrefix = "com.superwall.cache."
   static let ioQueuePrefix = "com.superwall.queue."
   static let defaultMaxCachePeriodInSecond: TimeInterval = 60 * 60 * 24 * 7 // a week
-
   static let instance = Cache(name: "default")
 
+  enum CacheKey: String {
```

Comment:
This looks awesome, I wonder if we could achieve even better type safety by including typing in these keys. Writing an array to `appUserId` shouldn't work but writing a string should.

Reply:
Been working on this and will be in the next push to this branch!